handbook/engineering/distribution/index.md

```diff
@@ -4,11 +4,15 @@
 
 The distribution team is responsible for making Sourcegraph easy to deploy, scale, monitor, and debug. We solve the most challenging problems our customers face deploying Sourcegraph on-premise in a variety of environments, and on [Sourcegraph.com](https://sourcegraph.com/search) - the largest Sourcegraph instance in the world.
 
-## Contact
+## Communication
```

Comment:
It seems `Contact` is a standard header in all the team pages, we might want to avoid changing this